src/app/services/items/index.ts

```diff
@@ -1,9 +1,30 @@
+import { Observable, Subject } from 'rxjs';
```

Comment:
for the sake of  CLI consider moving RX/ Saga services and imports to separate files, instead of keeping them as one

Reply:
no longer applying